local_server.sh

```diff
@@ -25,6 +25,9 @@ redis-server config/redis_local.conf &
 echo "Starting compass watch"
 compass watch server &
 
+echo "Starting jsx compiler"
+jsx --watch -x jsx server/static/jsx/ server/static/js/ &
```

Comment:
If we're using React, we'll also have to ensure developers are set up for it — that means, adding installation to `setup.sh` and other appropriate files (feel free to do this in another PR).